org.angryip.ipscan.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: org.angryip.ipscan
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: ipscan
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-config/gtk-3.0:ro;
```

Comment:
why is `xdg-config/gtk-3.0:ro` needed?

Reply:
this is not allowed by default, you will need an exception if it is actually necessary.

Reply:
Done, deleted.